.github/workflows/integ-tests.yml

```diff
@@ -0,0 +1,62 @@
+name: Integration Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  AWS_REGION : "us-west-2"
+  REGION : "us-west-2"
+  AMZN_BRAKET_OUT_S3_BUCKET : ${{ secrets.bucket_name }}
+  REPOSITORY_NAME : ${{ secrets.repository_name }}
+  ACCOUNT_ID : ${{ secrets.account_id }}
+  BUILD_RESULTS_PATH : "build_results.txt"
```

Comment:
Do we want to append a disambiguator or run id to these to avoid conflicting results?

Reply:
This file is only used locally within the build.